api/src/index.ts

```diff
@@ -120,14 +123,18 @@ app.post(
 
       return json(result)
     } catch (error) {
+      console.error('Payment quote creation error: ', {
+        error: error instanceof Error ? error.message : String(error),
+        stack: error instanceof Error ? error.stack : undefined
+      })
```

Comment:
```suggestion
      console.error('Payment quote creation error: ', JSON.parse(JSON.stringify(error, Object.getOwnPropertyNames(error))))
```

Reply:
Can extract this into a helper: `errorToJson`. This will get us all properties for that error, including `code` if it exists.